packages/info/README.md

```diff
@@ -9,22 +9,55 @@ This package returns a set of information related to the local environment.
 ## Installation
 
 ```bash
+#npm
 npm i -D @webpack-cli/info
+
+#yarn
+yarn add @webpack-cli/info -D
 ```
 
 ## Usage
 
+### Args / Flags
+
+#### Output format
+
+| Flag         | Description                   | Type        |     |     |
+| ------------ | ----------------------------- | ----------- | --- | --- |
+| `--json`     | To get the output as JSON     | [ boolean ] |     |     |
+| `--markdown` | To get the output as markdown | [ boolean ] |     |     |
+
+#### Options
+
+| Flag                   | Description                                                    | Type        |
+| ---------------------- | -------------------------------------------------------------- | ----------- |
+| `--help`               | Show help                                                      | [ boolean ] |
+| `--version`            | Show version number                                            | [ boolean ] |
+| `--system` , `--sys`   | System information ( OS, CPU )                                 | [ boolean ] |
+| `--binaries` , `--bin` | Installed binaries                                             | [ boolean ] |
+| `--browsers`           | Installed web browsers                                         | [ boolean ] |
+| `--npmg`               | Globally installed NPM packages ( webpack & webpack-cli only ) | [ boolean ] |
+| `--npmPackages`        | Info about webpack installed in the project                    | [ boolean ] |
+
 ### Node
 
 ```js
-const envinfo = require("@webpack-cli/info").default;
-envinfo();
+const info = require("webpack-cli/packages/info").default;
+
+async function wrapperFunc() {
+	let envinfo = await info({
```

Comment:
Is the config needed?

Reply:
Supplying a config from `--config` is great I think

Reply:
I meant the variable `envinfo`. Is it needed in this example?

Reply:
@evenstensberg shall I change the config input to  `--config=[PATH]` a flag?